monai/bundle/scripts.py

```diff
@@ -541,11 +541,15 @@ def load(
     return model
 
 
+@deprecated_arg_default("tag", "hosting_storage_v1", "dev", since="1.3", replaced="1.5")
 def _get_all_bundles_info(
     repo: str = "Project-MONAI/model-zoo", tag: str = "hosting_storage_v1", auth_token: str | None = None
 ) -> dict[str, dict[str, dict[str, Any]]]:
     if has_requests:
-        request_url = f"https://api.github.com/repos/{repo}/releases"
+        if tag == "dev":
+            request_url = f"https://raw.githubusercontent.com/{repo}/{tag}/models/model_info.json"
+        else:
+            request_url = f"https://api.github.com/repos/{repo}/releases"
```

Comment:
```suggestion
        if tag == "hosting_storage_v1":
            request_url = f"https://api.github.com/repos/{repo}/releases"
        else:
            request_url = f"https://raw.githubusercontent.com/{repo}/{tag}/models/model_info.json"
            
```
Could we do the check this way in case "tag" is actually for another tag in the repo?